5.5/root/usr/share/container-scripts/mysql/common.sh

```diff
@@ -52,16 +55,27 @@ function start_local_mysql() {
   wait_for_mysql $mysql_pid
 }
 
+# Shutdown mysql flushing privileges
+function shutdown_local_mysql() {
+  log_info 'Shutting down MySQL ...'
+  mysqladmin $admin_flags flush-privileges shutdown
+}
+
 # Initialize the MySQL database (create user accounts and the initial database)
 function initialize_database() {
-  echo 'Running mysql_install_db ...'
+  log_info 'Initializing database ...'
+  log_info 'Running mysql_install_db ...'
```

Comment:
Two lines are intentional. Initialization in this context is everything that is happening inside of the function. Running `mysql_install_db` is just part of it. We're also saying that initialization is finished when we're exiting the function.